### ISSUE TITLE
Form validation: Field 'Maximum of 2 words' Should be 'Minumum of 2 words'

### ISSUE THREAD

**Issue by NickelBomber** [wet-boew/wet-boew#7145]
The label for the field indicates that at most two words should be valid but the validator is actually  checking for at least 2 words.

See attached screenshot.

![capture](https://cloud.githubusercontent.com/assets/824532/10516555/cfc65782-7326-11e5-8ba9-e7e52c343e4e.PNG)

**Comment by pjackson28** [comment]
Good catch. Checking the source code below it, it is supposed to be "Minimum of 2 words". Somehow the actual example and the source code got out of sync.

```html
<div class="wb-frmvld">
	<form action="#" method="get" id="validation-example">
		...
		<div class="form-group">
			<label for="word2"><span class="field-name">Minimum of 2 words</span></label>
			<input class="form-control" id="word2" name="word2" type="text" data-rule-minWords="2" />
		</div>
```

**Comment by pjackson28** [comment]
Also checked the French and it is correct so it is only an issue with the label in the English example.

**Comment by pjackson28** [comment]
Will be fixed by PR #7149.
